NvAPIWrapper/Native/Display/BPC.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NvAPIWrapper.Native.Display
+{
+    public enum BPC
```

Comment:
Is this "Bits Per Color"? If so, the name should be changed to something that reflects that more directly. Like `HDRColorDepth` or something.